### PR TITLE
⚡ removing the b4mad related stuff #clarety

### PR DESCRIPTION
**Pull request by goern**
Signed-off-by: Christoph Görn <goern@b4mad.net>

### CODE REVIEW

**Comment by goern** [comment]
pre-commit warning arise from wrong comment indention

/cc @Gregory-Pereira

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Comment by goern** [comment]
@durandom wdyt?

**Comment by goern** [comment]
/kind feature
just testing...

File: grafana/overlays/moc/common/dashboards/b4mad/kustomization.yaml
Comment (tumido) [inline grafana/overlays/moc/common/dashboards/b4mad/kustomization.yaml:1]: This should also be removed from https://github.com/operate-first/apps/blob/master/grafana/overlays/moc/common/dashboards/kustomization.yaml

Comment (goern) [reply grafana/overlays/moc/common/dashboards/b4mad/kustomization.yaml:1]: in general? the whole dashboard configuration?

Comment (tumido) [reply grafana/overlays/moc/common/dashboards/b4mad/kustomization.yaml:1]: Well.. if I can see right, this PR is removing the file at

`grafana/overlays/moc/common/dashboards/b4mad/kustomization.yaml`

That means the reference in

[`grafana/overlays/moc/common/dashboards/kustomization.yaml#L14`](https://github.com/operate-first/apps/blob/0d1877d57f51e698d975646a125089a36f2e649f/grafana/overlays/moc/common/dashboards/kustomization.yaml#L14)

Points to a nonexistent location. And that means kustomize should fail to build `grafana/overlays/moc/common/dashboards`, which is not wanted probably...

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**